gadget/install/install.go

```diff
@@ -440,7 +440,7 @@ func onDiskVolumeFromPartitionSysfsPath(partPath string) (*gadget.OnDiskVolume,
 // applyLayoutToOnDiskStructure finds the on disk structure from a
 // partition node and takes the laid out information from laidOutVols
 // and inserts it there.
-func applyLayoutToOnDiskStructure(onDiskVol *gadget.OnDiskVolume, partNode string, laidOutVols map[string]*gadget.LaidOutVolume, gadgetVolName string) (*gadget.LaidOutStructure, error) {
+func applyLayoutToOnDiskStructure(onDiskVol *gadget.OnDiskVolume, partNode string, laidOutVols map[string]*gadget.LaidOutVolume, gadgetVolName string, creatingPart bool) (*gadget.LaidOutStructure, error) {
```

Comment:
the name and description of this seems somehow reversed now? something seems to need changing

Reply:
are there other cases like this where names/description don't match anymore because of the things we moved in the refactors?

Reply:
I've changed the function name now. I've taken a look and I've not seen more obvious cases, but it is not impossible that I overlooked some variable names that now are for different things.